README.md

```diff
@@ -1,10 +1,12 @@
 # Myth Buster 🤯
 
+## Idea - List of myths debunked over the last decade, with a focus on including funny myths
+
 ## Features ✌
 
 1. **User Vote/Ranking:**
    - Users can vote and rank myths based on their preferences.
-2. **Sort/Filter:**
+2. **Sort/Filter:**n
```

Comment:
There seems to be a typo in the Sort/Filter section of the README.md file. Please correct it. [important]